extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -191,6 +192,12 @@ describe('amp-a4a', () => {
     expect(child.src).to.match(/^https?:[^?#]+nameframe(\.max)?\.html/);
     const nameData = child.getAttribute('name');
     expect(JSON.parse.bind(null, nameData), nameData).not.to.throw(Error);
+    const attributes = JSON.parse(nameData);
```

Comment:
the only part of the data structure that I care about is that it has a ._context.sentinel, which I check below. I'll change to only parse once